oblt-cli/run/action.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Oblt-cli wrapper'
+description: 'Run the oblt-cli wrapper.'
+inputs:
+  command:
+    description: 'The oblt-cli command to run'
+    required: true
+  github-token:
+    description: 'The GitHub access token.'
+    required: true
+  slack-channel:
+    description: 'The slack channel to notify the status.'
+    default: '#observablt-bots'
+    required: false
+  username:
+    description: 'Username to show in the deployments with oblt-cli, format: [a-z0-9]'
+    default: 'obltmachine'
+    required: false
+runs:
+  using: "composite"
+  steps:
+    - uses: elastic/oblt-actions/oblt-cli/setup@v1.2.0
+      with:
+        github-token: ${{ inputs.github-token }}
+        slack-channel: ${{ inputs.slack-channel }}
+        username: ${{ inputs.username }}
+    - name: run oblt-cli
+      run: |
+        GITHUB_TOKEN=${{ inputs.token }} oblt-cli ${{ inputs.command }} --verbose --disable-banner
```

Comment:
`GITHUB_TOKEN=${{ inputs.github-token }}`

**nit**:
what's the reason to use this approach rather than:

```yaml
env:
  GITHUB_TOKEN: ${{ inputs.token }}
```

Reply:
Good question, but I don't remember why it was implemented this way in apm-pipeline-library.

https://github.com/elastic/oblt-actions/pull/12/commits/96a6899e6477e6724844bb12d231d57b82806148